### PR TITLE
fix(7.9 RN): add explicit limitation on edge

### PR DESCRIPTION
**Pull request by laurentleseigneur**
switch feature not working on edge

### CODE REVIEW

**Comment by tnguyen1** [comment]
retest this please

**Comment by tnguyen1** [comment]
retest this please

File: md/release-notes.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ For more information, read [the dedicated documentation](variables.md).
 
 #### Switch widget
 Directly from whiteboard or from widget properties section, you will now be able to change your current widget by a new one without loosing the configuration you have done. Useful when switching to a new version of your widget custom or not, and as first customization steps after your form creation from studio to match your end user needs.
+⚠️This feature is not available on Edge browser.
```
Comment (tbouffard) [inline md/release-notes.md]: ❓ is this working on IE11? usually, if something does not work on Edge, it doesn't work on IE 😸

Comment (laurentleseigneur) [reply md/release-notes.md]: good catch.

Is IE 11 still in the matrix ? i don't have this browser to test, but if same limitation is present, it should also be added of course

Comment (tbouffard) [reply md/release-notes.md]: yes IE11 is in the 7.9 matrix and for now, will also be in the 7.10 matrix
I am asking the UID Team about the IE limitation

Comment (benjaminParisel) [reply md/release-notes.md]: True, we need to put IE11 limitations too ! Good catch !

Comment (laurentleseigneur) [inline md/release-notes.md]:
```suggestion
⚠️This feature is not available on Edge and IE 11 browsers.  
```

**Review by tbouffard — APPROVED** [review]

**Pull request merged**